_config.yml

```diff
@@ -1 +1,2 @@
+website: https://geogeeks.org
```

Comment:
Should this be `baseurl`? https://jekyllrb.com/docs/configuration/options/#build-command-options

Reply:
No. Looks there is some inconsistent terminology between the `jekyll build` command and the site config file. Using `website` works, but `baseurl` doesn't. This is also outlined in the article linked above.

Reply:
A bit confusing 😅

Reply:
ah right! cool.